src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    public static final Injector injector = Injector.getInstance("mate.academy");
+    public static final BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
     public static void main(String[] args) {
+        Book firstBook = new Book();
+        firstBook.setTitle("firstBook");
+        firstBook.setPrice(50);
+
+        Book secondBook = new Book();
+        secondBook.setTitle("secondBook");
+        secondBook.setPrice(70);
```

Comment:
The `setPrice` method is currently using an `int` value. Ensure that the `Book` model's `price` field is of type `BigDecimal` and update this line to use `BigDecimal.valueOf(70)` instead.